### PR TITLE
Promote `flutter_svg` and `rohd_vf` to `stable`

### PR DESCRIPTION
**Pull request by domesticmouse**
Fixes: https://github.com/dart-lang/dart-pad/issues/2447
Fixes: https://github.com/dart-lang/dart-pad/issues/2461

### CODE REVIEW

File: tool/pub_dependencies_stable.json [hunk 1]
```diff
@@ -112,6 +114,7 @@
   "riverpod_navigator": "1.0.10",
   "riverpod_navigator_core": "1.0.10",
   "rohd": "0.4.2",
+  "rohd_vf": "0.4.1",
```
Comment (parlough) [inline tool/pub_dependencies_stable.json:117]: Should the `beta` and `master` versions of this file be updated with these too?

Comment (domesticmouse) [reply tool/pub_dependencies_stable.json:117]: Yeah, I was cheating by just cut'n'pasting the `dev` file over `stable`. I need to run the toolchains to update everything else, which thanks to I/O I don't really have time to do today. I'll definitely being doing it in a week or so when the next `stable` hotfix or `beta` lands and forces me to regenerate the precompiled SDK upload blobs.

**Review by parlough — APPROVED** [review]
Looks good to me after testing on the `dev` channel, cool to get these in. 

I regenerated the dependencies in https://github.com/dart-lang/dart-services/pull/1045 if you prefer to land updates for all channels at once. Either is fine.

**Comment by domesticmouse** [comment]
Closing out in deference to @parlough's better PR

**Pull request closed without merging**